mitodl/mitxonline
# Convert grade into letter for program records

#### Pre-Flight checklist


#### What are the relevant tickets?
Fix https://github.com/mitodl/mitxonline/issues/1647

#### What's this PR do?
Populates all passing grades missing letter grade before Fall 2022 semester by a fixed formula.

#### How should this be manually tested?
Create a grade for your user. Go to the program record view to see how it is displayed.
Make sure there is a passing grade for a course run that ended before Sept 1st 2022, where letter_grade is set to 'Pass'
Then run migrations and make sure that the letter_grade was assigned a letter.
<img width="1299" alt="Screen Shot 2023-06-05 at 2 00 35 PM" src="https://github.com/mitodl/mitxonline/assets/7574259/0c647a3a-332f-42fc-9e55-58e9e5dfdf82">
